packages/flight/index.js

```diff
@@ -0,0 +1,467 @@
+            text-align: center;
+            font-size: 16px;
+            color: #9aa0a6;
+        }
+        
+        #presearch-flight-package .flight-details {
+            display: flex;
+            margin-top: 10px;
+            margin-bottom: 10px;
+            flex-wrap: wrap;
+        }
+
+        #presearch-flight-package .flight-detail {
+            flex: 1;
+        }
+
+        #presearch-flight-package .flight-detail-row {
+            display: flex;
+        }
+
+        #presearch-flight-package .flight-detail-row > div {
+            margin-right: 20px;
+        }
+
+        #presearch-flight-package .flight-detail-label {
+            color: #7e8184
+        }
+        
+        .dark #presearch-flight-package .flight-detail-label {
+            color: #9aa0a6;
+        }
+
+        #presearch-flight-package .flight-detail-row > div > div:not(.flight-detail-label) {
+            font-size: 22px;
+        }
+        
+        #presearch-flight-package .flight-detail-scheduled-time {
+            color: #4eb66e;
+            font-size: 22px;
+            width: 100px;
+        }
+        
+        #presearch-flight-package .flight-detail-estimated-time {
+            display: none;
+            width: 100px;
+        }
+        
+        #presearch-flight-package .flight-not-in-time .flight-detail-scheduled-time {
+            color: #9aa0a6;
+            font-size: 14px !important;
+            text-decoration: line-through;
+        }
+        
+        #presearch-flight-package .flight-not-in-time .flight-detail-estimated-time {
+            color: #4eb66e;
+            font-size: 22px;
+            display: block;
+        }
+
+        #presearch-flight-package .flight-detail .flight-point-city {
+            display: none;
+        }
+
+        #presearch-flight-package .flight-detail .flight-point-city::after {
+            content: ' ·'
+        }
+        
+        @media only screen and (max-width: 600px) {
+            #presearch-flight-package .flight-detail .flight-point-city {
+                display: inline;
+                color: inherit;
+                font-weight: bold;
+            }
+        }
+
+        @media only screen and (min-width: 768px) {
+            #presearch-flight-package {
+                width: 554px;
+            }
+        }
+        
+        @media only screen and (min-width: 1024px) {
+            #presearch-flight-package {
+                width: 644px;
+            }
+        }
+
+    </style>`;
+}
+
+async function getFlight(query, API_KEY) {
+    const flightNo = extractFlightNo(query);
+
+    if (!flightNo) {
+        return null;
+    }
+
+    try {
+        const flightUrl = `${FLIGHT_API_URL}/flights`;
+        const { data } = await axios.get(flightUrl, {
+            params: {
+                "access_key": API_KEY,
+                "flight_iata": flightNo
+            }
+        });
+
+        const flights = data.data;
```

Comment:
We need to catch request error here 
```suggestion
        const request = await axios.get(flightUrl, {
            params: {
                "access_key": API_KEY,
                "flight_iata": flightNo
            }
        }).catch(error => ({ error });;

        if (request.error) {
            return null;
        }
        const flights = request.data.data;
```

Reply:
Code is inside try/catch block which will catch error if such.